# QSAR Modelling of Pan-Coronavirus Antivirals Small Molecule Candidates 
This repository is the culmination of collaborative research on COVID-19 with researchers from the University of Antioquia and the Jenner Institute, University of Oxford. Research is taking on a new direction, prototyping computational biophysics algorithms that facilitate the development of pan-coronavirus antivirals. Reflecting our research on evaluating and advancing the current state-of-the-art in AI - e.g. generative biology - this repository also complements our work on developing globally accessible novel COVID-19 antiviral therapeutics.
### COVID Moonshot
With global vaccine inequality and the rapid spread of variants of concern, developing safe, globally accessible and affordable COVID-19 antiviral therapeutics is imperative. [COVID Moonshot](https://covid.postera.ai/covid), an open drug discovery consortium, was created to tackle this challenge and is supported by the [COVID-19 Therapeutics Accelerator](https://www.therapeuticsaccelerator.org/). This repository aims to contribute to these open-science efforts, including the [COVID R&D Alliance](https://www.covidrdalliance.com/).
### Pan-Coronavirus Antivirals
This repository explores inhibitors that could work on the family of betacoronaviruses, comprising SARS, MERS and COVID-19 - to find a pan-betacoronavirus inhibitor. Theoretically, we can thwart families of viruses by targeting their related highly conserved features (viral targets) or interfering with human pathways that viruses generally exploit to infect cells (host targets). We're formulating strategies informed by heterologous immunity. Identifying common denominators of coronaviruses for drug targets and small molecule antiviral candidates can help us to develop a  pan-betacoronavirus inhibitor.
### References
1. Rona, G., Zeke, A., Miwatani-Minter, B. et al. The NSP14/NSP10 RNA repair complex as a Pan-coronavirus therapeutic target. Cell Death Differ 29, 285–292 (2022). [https://doi.org/10.1038/s41418-021-00900-1](https://www.nature.com/articles/s41418-021-00900-1#citeas)
2. Bazaga, A., Leggate, D. & Weisser, H. Genome-wide investigation of gene-cancer associations for the prediction of novel therapeutic targets in oncology. Sci Rep 10, 10787 (2020). [https://doi.org/10.1038/s41598-020-67846-1](https://www.nature.com/articles/s41598-020-67846-1#citeas)
3. Brendan T. Freitas, Daniil A. Ahiadorme, Rahul S. Bagul, Ian A. Durie, Samir Ghosh, Jarvis Hill, Naomi E. Kramer, Jackelyn Murray, Brady M. O’Boyle, Emmanuel Onobun, Michael G. Pirrone, Justin D. Shepard, Suzanne Enos, Yagya P. Subedi, Kapil Upadhyaya, Ralph A. Tripp, Brian S. Cummings, David Crich, and Scott D. Pegan ACS Infectious Diseases 2022 8 (3), 596-611 https://doi.org/10.1021/acsinfecdis.1c00631
# Our Approach
Use AI-ML to build mechanistic mathematical models of gene regulatory networks and signalling pathways that support the research of antiviral therapeutics. Provide quantitative, mechanistic and predictive insights into understanding the disease biology and human immunological responses to beta-coronaviruses. Models help us to:

* Identify the regulation of immune responses to beta-coronaviruses: understand GRNs immune response mechanisms; identify immune response biomarkers - networks as biomarkers. 
* Make predictions about the causal regulatory networks controlling immunological responses: in other words, predict regulatory network responses to antivirals, which are likely to activate several innate immune pathways in parallel. 
* Understand differences in immunological responses through comparative network analysis. Explore why some people are asymptomatic, develop mild or severe symptoms, or succumb to the fatality of beta-coronaviruses.  

With the threat of pandemics omnipresent, we need to build the capability of developing antiviral therapeutics quickly. Quantitative biology AI-ML formulated in the science of complex systems can help us. 






